src/styles/stars.scss

```diff
@@ -0,0 +1,18 @@
+.stars {
+  @include flexbox-between;
+
+  &__star {
+    background: url(../images/star.svg) center no-repeat;
+    height: 16px;
+    width: 16px;
+    border-radius: 0.5px;
+
+    &:not(:last-child) {
+      margin-right: 4px;
+    }
+
+    &:nth-child(-n+4) {
+      background-image: url(../images/star-active.svg);
+    }
```

Comment:
The amount of active stars should change by changing modifier in html(see comment in html). Reuse styles from `stars` task.

Reply:
+